config/hello_client.json

```diff
@@ -5,7 +5,7 @@
   "logging": {
     "level": "debug",
     "console": "true",
-    "file": { "enable": "false", "path": "/var/log/vsomeip.log" },
+    "file": { "enable": "false", "path": "/tmp/vsomeip.log" },
```

Comment:
is there a reason why log file path is updated only in hello_client.json from "/var/log/vsomeip.log" to "/tmp/vsomeip.log" but not in hello_service.json and hello_client-proxy.json ?

Reply:
No I just committed my work in progress, generally tmp is better for logging as it does not require root accesa. Feel free to sync configs.